samples/sample_full_pbr.cpp

```diff
@@ -76,13 +77,13 @@ struct PbrMap {
 };
 
 static std::array<PbrMap, MAP_COUNT> g_maps = {
-    PbrMap { "_color",      "baseColorMap",  true,  nullptr },
-    PbrMap { "_ao",         "aoMap",         false, nullptr },
-    PbrMap { "_roughness",  "roughnessMap",  false, nullptr },
-    PbrMap { "_metallic",   "metallicMap",   false, nullptr },
-    PbrMap { "_normal",     "normalMap",     false, nullptr },
-    PbrMap { "_bentNormal", "bentNormalMap", false, nullptr },
-    PbrMap { "_height",     "heightMap",     false, nullptr },
+    PbrMap { "albedo",     "baseColorMap",  true,  nullptr },
```

Comment:
baseColor would be fine but I strongly dislike albedo as a term :/

Reply:
Also _color is what's used in the sample sets in third_party/textures/

Reply:
it looks like we're using different terms in different places :-(

Reply:
Where do we use albedo? I've always avoided that term. It's never mentioned in the docs, etc.

Reply:
okay, I've switched back to "color" and renamed monkey/albedo.png to monkey/color.png